src/main/java/com/bobcat00/altdetector/Config.java

```diff
@@ -26,6 +26,20 @@ public class Config
 {
     private AltDetector plugin;
     
+    // Constants for configuration keys
```

Comment:
I don't know how to do reviews with Github properly. But this file has multiple errors: You removed saving several config sections, you changed some strings that shouldn't be changed, changed replaceAll to replace, you save the same comment multiple times, and you removed a call to saveConfig. I suggest you go back to the original file and simply add the parts needed for the Discord changes.